src/components/SearchModal/CurrencySearch.tsx

```diff
@@ -104,20 +104,24 @@ export function CurrencySearch({
   const filteredSortedTokens = useSortTokensByQuery(debouncedQuery, sortedTokens)
 
   const native = useNativeCurrency()
+  const wrapped = native.wrapped
```

Comment:
Is there a case (celo?) where `native.equals(native.wrapped)`? In this case, you should omit `wrapped` (ie `const wrapped: Token | undefined = native.wrapped.equals(native) ? undefined : native.wrapped`).

Reply:
it looks like this works in celo in the preview, but I'm not sure how...
maybe it uses disableNonToken?